python/hsfs/feature_view.py

```diff
@@ -211,9 +215,19 @@ def init_serving(
         training_dataset_version: Optional[int] = None,
         external: Optional[bool] = None,
         options: Optional[dict] = None,
+        init_online_store_sql_client: Optional[bool] = None,
```

Comment:
What is the use case of supporting both client in a single feature view instance? It seems that the motivation of using rest client is that sql client is not available. Alternatively, users can create 2 fv instance and `init_serving` with different clients.